docs/support.html

```diff
@@ -0,0 +1,151 @@
+        <h2 class="text-center">Support</h2>
+        <h3>Frequently Asked Questions</h3>
+        <div>
+          <h5>My download seems frozen. What should I do? 🧊</h5>
+          <p>
+            If you really thing your download is frozen, restart the download
+            with the "Show progress updates" (under advanced settings) enabled.
+            This will show you exactly how many files have been downloaded at
+            each step.
+          </p>
+
+          <h5>Why does it keep saying my input file is expired? 🌧</h5>
+          <p>
+            This happens when ALL of the URL's found in the
+            <tt>memories_history.json</tt> file return a bad response from
+            Snapchat's servers. The <tt>memories_history.json</tt> file only
+            contains valid URL's for approximately 7 days after downloading your
+            data from Snapchat. To verify that your download file is still valid
+            watch
+            <a
+              class="text-dark"
+              href="https://youtu.be/fDc3pZ9guCg"
+              title="Check if memories_history.json file is expired"
+              >this video</a
+            >.
+          </p>
+
+          <h5>Why does it say some of my memories failed to download? 🤔</h5>
+          <p>
+            Something went wrong when trying to fetch some of your files. The
+            application attempted to download those memories twice, but failed
+            both times. This is likely because something went wrong on
+            Snapchat's side. When this happens the application shows you the
+            time and type of memory so that you can manually save these files.
+          </p>
+
+          <h5>Does the application do anything with my data? 🔐</h5>
+          <p>
+            Your data is only used for requesting your files from Snapchat's
+            servers. Your data is used to communicate with Snapchat's servers
+            directly and is not used in any other way.
+          </p>
+
+          <h5>Can I do this on my phone? 📱</h5>
+          <p>No. The application only runs on computers.</p>
+
+          <h5>Which version should I download if I'm using a Chromebook? 💻</h5>
+          <p>
+            Download the linux version of the application. That is the most
+            likely to work on Chrome OS.
+          </p>
+        </div>
+
+        <br />
+        <p>
+          Have a question you don't see listed above or just want to provide
+          some feedback? Feel free to reach out to the creator via the contact
+          button below.
+        </p>
+
+        <a href="https://www.buymeacoffee.com/cal.overflow" target="_blank"
+          ><button class="btn btn-dark my-2">
+            <img src="bmc.png" width="18" height="30" />
+            <span class="align-middle">Support creator</span>
+          </button></a
+        >
+        <a
+          href="http://www.cal-overflow.dev/contact?memoryDownload=true"
```

Comment:
It looks like your website has encryption, so I imagine we want to use `https` for this

Reply:
I always forget to include this 🤦 \
http://cal-overflow.dev enforces HTTPS connection so this is alright to leave for now.